docs/quickstart.rst

```diff
@@ -0,0 +1,163 @@
+
+.. code-block:: bash
+
+   ethpm activate ethpm://packages.ethpm.eth/package@1.0.0
+
+If you have authenticated a wallet account with ``ethpm auth``, pass your password in with the ``--keyfile-password`` flag to automatically configure all contract factories, deployments and web3 instances to sign for your account.
+
+.. code-block:: bash
+
+   ethpm activate ethpm://packages.ethpm.eth/package@1.0.0 --keyfile-password xxx
+
+To instantly interact with any verified contract on Etherscan, use an `Etherscan URI <https://docs.ethpm.com/uris#etherscan-uris>`__ (though, this will require you setting you Etherscan API key to the environment variable: ``ETHPM_CLI_ETHERSCAN_API_KEY``).
+
+.. code-block:: bash
+
+   ethpm activate etherscan://0x123c0ntractaddr3ss890:1
+
+
+Install
+-------
+
+The `install <commands.html#install>`__ command will install any ethPM package to a local ``_ethpm_packages/`` directory. Think of this directory like ``node_modules/`` in npm. The files are written to disk according to `this scheme <disk.html>`__. By default, ``ethpm install`` will look for an ``_ethpm_packages/`` in the current working directory, but a specific ``_ethpm_packages/`` directory can be targeted if you pass in its path with the ``--ethpm-dir`` flag. If you want to install a package under an alias, you can use the ``--alias`` flag to do so. If you're installing an etherscan verified contract as a package, you **must** pass in ``--package-name`` and ``--package-version`` flags.
+
+.. code-block:: bash
+
+   ethpm install ethpm://packages.ethpm.eth/package_name@1.0.0
+
+List all installed packages.
+
+.. code-block:: bash
+
+   ethpm list
+
+Uninstall a package.
+
+.. code-block:: bash
+
+   ethpm uninstall package_name
+
+
+Create
+------
+
+To create your own ethPM package from local contracts requires compilation. If you don't have the `Solidity Compiler <https://solidity.readthedocs.io/en/v0.5.13/installing-solidity.html>`__ installed on your machine, there are `frameworks available <https://docs.ethpm.com/ethpm-developer-guide/ethpm-core-libraries>`__ to help with the compilation and automatically generate your ethPM package.
+
+If you have the Solidity compiler installed on your machine, the best way to get started is with the manifest wizard. The wizard expects a project directory with the following structure. 
+
+- project/
+  - contracts/
+    - xxx.sol
+    - yyy.sol
+
+Pass in a path to your project directory under the ``--project-dir`` flag. The wizard will attempt to compile these contracts using the available ``solc`` on your machine. Then it will prompt you for package details.
+
```

Comment:
Do you think it's worth documenting a minimum required version? This way a user knows that the package they want to release must use the same pragma as the `registry` if they are to deploy their own.

Reply:
Good call. Just to clarify though, the minimum solc required version would just be whatever version is required to compile the contracts that live in the project directory - not the pragma required for the registry contract. The registry contract is already compiled and deployed to the blockchain - to release a package on the registry all you need is the `package_name`, `version` and `manifest_uri`